mindsdb/integrations/handlers/rag_handler/requirements.txt

```diff
@@ -2,9 +2,8 @@
 faiss-cpu
 openai==1.6.1
 html2text
-sentence-transformers
 writerai~=1.1.0
 pydantic
-sentence-transformers
-faiss-cpu
 langchain-community  # for Writer LLM integration
+sentence-transformers # needed for HuggingFaceEmbeddings from langchain-community
```

Comment:
@QuantumPlumber it fails without sentence-transformers. Adding back in.

Also updated unit test as they were failing as they hadn't been updated along with most recent code changes i.e. 'input_column' is non manidatory

 see https://github.com/langchain-ai/langchain/blob/b20c2640dac79551685b8aba095ebc6125df928c/libs/community/langchain_community/embeddings/huggingface.py#L65

Reply:
This is only required if the user wants to use hugging face embeddings. I'm arguing for removing that support as it is not required for 3rd party LLM integration.

Reply:
hugging face embedings (sentence-transformers) is the only option for embeddings in rag_handler. Therefore sentence-transformers is required.

Reply:
Oh.. so this will probably be cleaned up when RAG is consolidated.